src/components/ShipToCustomerModal.vue

```diff
@@ -14,34 +14,32 @@
     <main>  
       <ion-item>
         <ion-label position="fixed">{{ $t("First name") }}</ion-label>
-        <ion-input />
+        <ion-input v-model="firstName" />
       </ion-item>
       <ion-item>
         <ion-label position="fixed">{{ $t("Last name") }}</ion-label>
-        <ion-input />
+        <ion-input v-model="lastName" />
       </ion-item>
       <ion-item>
         <ion-label position="fixed">{{ $t("Street") }}</ion-label>
-        <ion-input />
+        <ion-input v-model="street" />
       </ion-item>
       <ion-item>
         <ion-label position="fixed">{{ $t("City") }}</ion-label>
-        <ion-input />
+        <ion-input v-model="city" />
       </ion-item>
       <ion-item>
         <ion-label position="fixed">{{ $t("State") }}</ion-label>
-        <ion-input />
+        <ion-input v-model="state" />
       </ion-item>
       <ion-item>
         <ion-label position="fixed">{{ $t("Zipcode") }}</ion-label>
-        <ion-input />
+        <ion-input v-model="zipcode" />
       </ion-item>
       <ion-item>
         <ion-label>{{ $t("Shipping method") }}</ion-label>
-        <ion-select value="next-day">
-          <ion-select-option value="same-day">Same day</ion-select-option>
-          <ion-select-option value="next-day">Next day</ion-select-option>
-          <ion-select-option value="second-day">After 2 days</ion-select-option>
+        <ion-select :value="shipmentMethod" :selected-text="getShipmentMethod(shipmentMethod)" @ionChange="changeShipment($event)">
```

Comment:
Why we have used value and selected-text? We just need to use `:value` and not both.

Reply:
Sir, we are using here prop to pass data from parent **orders.vue** component to modal which can not be directly bound with **:value** attribute of ion-select that's why we have used another data property which is rendering after UI renders so that we are not showing the value of the selected property but showing selected text with the help of selected-text.